src/style.css

```diff
@@ -0,0 +1,147 @@
+body {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  background: #e5e5e5;
+}
+
+:root {
+  --fonovy-color: #fff;
+  --button-color: #00acdc;
+  --secondary_text-color: #616070;
+  --main_text-color: #060b35;
+}
+
+.conteiner {
+  width: 200px;
+  height: 408px;
+}
+
+.Product {
```

Comment:
add padding here for whole card, dont need use margin for every child block for position in card and use flex for centered all element